### PR TITLE
Make Prime Minister dynamic on "How government works" page

### PR DESCRIPTION
**Pull request by alexmuller**
This is very much a first stab at this. Be gentle, I don't write code anymore. If it looks vaguely ok I'll write a test for it and get the other tests passing.

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def feed
   def how_government_works
     sitewide_setting = load_reshuffle_setting
     @is_during_reshuffle = sitewide_setting.on if sitewide_setting
+    @prime_minister = MinisterialRole.where(slug: 'prime-minister').first.current_person
```
Comment (alexmuller) [inline app/controllers/home_controller.rb]: This feels a bit naive, is there a better way?

Comment (tijmenb) [inline app/controllers/home_controller.rb]: `MinisterialRole.find_by(slug: 'prime-minister').current_person` would be slightly better perhaps, to make it clear that the slug is unique.

**Comment by tijmenb** [comment]
I think this looks fine 👍

File: test/functional/home_controller_test.rb [hunk 2]
```diff
@@ -57,18 +67,24 @@ class HomeControllerTest < ActionController::TestCase
...
 
+  view_test "how government works shows the current prime minister" do
+    get :how_government_works
+
+    assert_select '.prime-minister p a', 'Forename Surname'
+  end
+
   view_test "how government works page shows a count of cabinet ministers, other ministers and total ministers" do
-    david_cameron = create(:person, forename: 'David', surname: 'Cameron')
+    # david_cameron = create(:person, forename: 'David', surname: 'Cameron')
```
Comment (andrewgarner) [inline test/functional/home_controller_test.rb]: We shouldn't be committing commented out code.

Comment (alexmuller) [inline test/functional/home_controller_test.rb]: Yeah, I need a hand getting the tests to pass before this can be merged

Comment (gpeng) [inline test/functional/home_controller_test.rb]: Happy to pair on this if I can help.

**Comment by alexmuller** [comment]
@gpeng: Thanks, I think I'm almost there. The final(ish) thing that's confusing me is that the "Unpublishing a document and redirecting" feature is creating a FactoryGirl publication which is somehow going through the `HomeController#how_government_works` method.

**Comment by gpeng** [comment]
It's coincidentally using that page in the feature here https://github.com/alphagov/whitehall/blob/master/features/step_definitions/unpublishing_published_documents_steps.rb#L90 but the there's no `MinisterialRole` being setup within the feature so you're getting the no method on `nil` thing. You could probably just choose another page for that step.

**Comment by alexmuller** [comment]
@gpeng: Ooh thanks! Tests are passing now so this is good for somebody to have a look at.

**Comment by tijmenb** [comment]
👍 looks good to me (again)

**Pull request merged**